bin/node/runtime/voter-bags/src/main.rs

```diff
@@ -0,0 +1,68 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2021 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Make the set of voting bag thresholds to be used in `voter_bags.rs`.
+
+use pallet_staking::voter_bags::make_bags::generate_thresholds_module;
+use pallet_staking::mock::Test;
+use std::path::{Path, PathBuf};
+use structopt::{StructOpt, clap::arg_enum};
+
+arg_enum!{
+	#[derive(Debug)]
+	enum Runtime {
+		Node,
+		StakingMock,
+	}
+}
+
+impl Runtime {
```

Comment:
placing this here would inevitable mean that you cannot re-use it in polkadot, since we don't have this crate in polkadot. I recommend a simple version that just lives in `pallet-staking` or sth. 

You can address it later.

Reply:
Yeah, probably the ideal solution is to move this little helper into its own repo so it can depend on (and produce bag threshold lists for) polkadot, substrate node, etc.

The other option I was considering was to just copy the crate wholesale into `polkadot/runtime/common` and make appropriate modifications there. It would mean we'd have separate binaries, but it would also mean that each one could be simpler.